src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Trunc.cs

```diff
@@ -79,7 +79,7 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
                 if (type0.IsTable)
                 {
                     // Ensure we have a one-column table of numerics
-                    fValid &= CheckNumericColumnType(type0, args[0], errors, ref nodeToCoercedTypeMap);
+                    fValid &= CheckNumericColumnType(type0, args[0], context.Features, errors, ref nodeToCoercedTypeMap);
```

Comment:
With Greg's PR,  you are going to run into a conflict for signature of bunch of Check*Type functions.

Reply:
Yep, I've been resolving lots of conflicts with this PR :-(